accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -161,6 +538,39 @@ Windows-specific:
 * **PipeTransmissionMode**
     - `Message`
 
+### System.Media
+
+* **SoundPlayer**
```

Comment:
Should we mark the entire type?

Reply:
@dotnet/dotnet-winforms @JeremyKuhne

Reply:
I would presume so, why doesn't `System.Windows.Extensions` get excluded by default? @ericstj ?

Reply:
Agreed,  all of System.Windows.Extensions is currently windows-specific.

Reply:
Marked